staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -56,13 +56,6 @@ const (
 
 	// defaultBurst is the default burst to be used with the discovery client's token bucket rate limiter
 	defaultBurst = 300
-
-	AcceptV1 = runtime.ContentTypeJSON
-	// Aggregated discovery content-type (currently v2beta1). NOTE: Currently, we are assuming the order
-	// for "g", "v", and "as" from the server. We can only compare this string if we can make that assumption.
-	AcceptV2Beta1 = runtime.ContentTypeJSON + ";" + "g=apidiscovery.k8s.io;v=v2beta1;as=APIGroupDiscoveryList"
```

Comment:
since we'll be picking this back at least to 1.27 (and maybe 1.26? I can't remember when this got added), I would recommend structuring this to minimize code movement and avoid changing exported variables/constants